client/src/index.jsx

```diff
@@ -3,17 +3,116 @@ import ReactDOM from 'react-dom';
 
 import axios from 'axios';
 
+import RelatedItemList from './components/RelatedItemList.jsx';
+
+
 class App extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            relatedItems: [],
+            relatedItemInfo: [],
+            relatedItemsCurrent: [],
+            currentArr: 0,
+            maxRelatedLength: 5,
+            itemId: 40
+        }
+
+        // click bindings
+        this.arrowClick = this.arrowClick.bind(this);
+        this.itemClick = this.itemClick.bind(this);
+    }
+
+    arrowClick(dir) {
```

Comment:
A cleaner way to implement this function that changes the `this.state.currentArr` number can be the following: 
`
if (dir === 'left') {
  this.setState({
    currentArr: (this.state.currentArr + 1) % this.state.relatedItemsCurrent.length
  });
} else if (dir === 'right') {
  this.setState({
    currentArr: (this.state.currentArr - 1) > 0 ? (this.state.currentArr - 1) : (this.state.relatedItemsCurrent.length)
  });
}
`
instead of having to consider the edge cases explicitly.